source/common/http/utility.h

```diff
@@ -438,9 +438,7 @@ const std::string& getProtocolString(const Protocol p);
 
 /**
  * Return the scheme of the request.
- * For legacy code (envoy.reloadable_features.correct_scheme_and_xfp == false) this
- * will be the value of the X-Forwarded-Proto header value. By default it will
- * return the scheme if present, otherwise the value of X-Forwarded-Proto if
+ * By default it will return the scheme if present, otherwise the value of X-Forwarded-Proto if
```

Comment:
Will it still return X-Forwarded-Proto? Doesn't this PR changes it to only return the Scheme?

Or does this comment refer to this code:
https://github.com/envoyproxy/envoy/blob/a1f0d429fcd520497df40182adaac5efaaacb338/source/common/router/router.cc#L84-L91

Reply:
yeah the prior comment was incorrect thanks.  I'll go remove the utility now that it's a wrapper around getSchemeValue